src/main/java/org/opentestsystem/ap/imrt/common/service/OperationalEventServiceLoggerImpl.java

```diff
@@ -27,20 +27,21 @@ public void itemEvent(Logger logger, ItemEventType type, Integer key, String id,
     /**
      * Appends a message to a string builder and corresponding value to an argument list,
      * only if the value is non-null
-     * @param args  Existing list of arguments to append to
-     * @param builder  Existing string builder to append to
-     * @param message  Message to append, if value is non-null
-     * @param value  Value to append, if it is non-null
+     *
+     * @param args    Existing list of arguments to append to
+     * @param builder Existing string builder to append to
+     * @param message Message to append, if value is non-null
+     * @param value   Value to append, if it is non-null
      */
-    private void append(List<Object> args, StringBuilder builder, String message, Object value){
-        if(value != null) {
+    private void append(List<Object> args, StringBuilder builder, String message, Object value) {
+        if (value != null) {
             builder.append(message);
             args.add(value);
         }
     }
 
     @Override
-    public void serviceEvent(Logger logger, ServiceEventType type, Exception e, String format, Object... args) {
+    public void serviceEvent(Logger logger, ServiceEventType type, Throwable e, String format, Object... args) {
```

Comment:
The argument type is the only thing that changed.  Formatting in this class didn't follow IntelliJ defaults which is why there are formatting changes.